common/app/views/fragments/analytics/google.scala.html

```diff
@@ -22,6 +36,55 @@
             .map(function(xs) { return xs[1] })[0];
     }
 
+    @* copied from common/util/cookies.js *@
+    function getCookieValues(name) {
+        var cookieVals = [],
+            nameEq = name + '=',
+            cookies = document.cookie.split(';');
+
+        cookies.forEach(function (cookie) {
+            while (cookie.charAt(0) === ' ') {
+                cookie = cookie.substring(1, cookie.length);
+            }
+
+            if (cookie.indexOf(nameEq) === 0) {
+                cookieVals.push(cookie.substring(nameEq.length, cookie.length));
+            }
+        });
+
+        return cookieVals;
+    }
+
+    @* copied from common/util/cookies.js *@
+    function getCookie(name) {
+        var cookieVal = getCookieValues(name);
+
+        if (cookieVal.length > 0) {
+            return cookieVal[0];
+        } else {
+            return null;
+        }
+    }
+
+    @* copied from common/modules/identity/api.js *@
+    function decodeBase64(str) {
+        return decodeURIComponent(escape(utilAtob(str.replace(/-/g, '+').replace(/_/g, '/').replace(/,/g, '='))));
+    }
+
+    @* based on common/modules/identity/api.js *@
+    function getIdentityIdFromCookie() {
+        var cookieData = getCookie('GU_U'),
+            userData = cookieData ? JSON.parse(decodeBase64(cookieData.split('.')[0])) : null;
+        if (userData) {
+            return userData[0];
+        } else {
+            return null;
+        }
+    }
+
+    var identityId = getIdentityIdFromCookie();
+    var isLoggedIn = (identityId !== null);
```

Comment:
@ajosephides Is this what you intended for the `isLoggedIn` dimension? Just a flag to say whether the cookie is set or not? Seems a little redundant.